_data/projects/boa.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Boa
+desc: Javascript Engine written in Rust
+site: https://github.com/jasonwilliams/boa
+tags:
+  - rust
+  - oss
+  - javascript
+  - language
+  - compiler
+  - interpreter
+upforgrabs:
+  name: good first issue
+  link: https://github.com/jasonwilliams/boa/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue"
+stats:
```

Comment:
No need to include these next few lines - the site will fall back to fetching new stats from the API initially, and after this is merged a new Action run will add in the stats

Reply:
removed